modules/dataplex-datascan/versions.tf

```diff
@@ -17,11 +17,11 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 5.34.0, < 6.0.0" # tftest
+      version = ">= 5.37.0, < 6.0.0" # tftest
```

Comment:
bump `default-versions.tf` in the root of the repo, then run `find . -name versions.tf -exec cp default-versions.tf {} \;` to bump all module versions